manifests/init.pp

```diff
@@ -75,7 +75,7 @@
   $default_app_id      = $::kibana::params::default_app_id,
   $request_timeout     = $::kibana::params::request_timeout,
   $shard_timeout       = $::kibana::params::shard_timeout,
-  $legacy_service_mode = $::kibana::params::legacy_service_mode,
+  $service_provider    = $::kibana::params::service_provider,
```

Comment:
Actually I don't need this in init.pp since it's not going to be set by the users. I can reference it directly using ```::kibana::params::service_provider```.